newIDE/app/src/MainFrame/EditorContainers/HomePage/LearnSection/MainPage.js

```diff
@@ -97,8 +95,9 @@ const MainPage = ({
   tutorials,
 }: Props) => {
   const classes = useStyles();
+  const { isFlowRunning } = React.useContext(InAppTutorialContext);
```

Comment:
should we give this prop a more specific name like `isInAppTutorialRunning`?

`isFlowRunning` is a bit too abstract without the context

Reply:
Good idea!